data.json

```diff
@@ -27,6 +27,31 @@
 		"url_regex": "https%3A%2F%2Fwww.porntrex.com%2Fvideo%2F.*%2F.*",
 		"video_xpath": "//div[@id='kt_player']/div[contains(@class, 'player')]/video"
 	},
+	{
+		"url_regex": "https%3A%2F%2Fwww.povaddict.com%2F.*",
+		"video_xpath": "//video[@id='video-player_html5_api']"
+	},

```

Comment:
Matches the entire site, causing errors.
Should only match videos like:
https://www.povaddict.com/\\d+/.*
Since each video is behind a number, then a title
'https%3A%2F%2Fwww.povaddict.com%2F%5Cd%2B%2F.*'